Integrations/McAfee_ESM-v10/McAfee_ESM-v10.py

```diff
@@ -904,6 +882,7 @@ def get_minutes_left():
                 triggered_date = alarm['triggeredDate']
                 if next_run is None or next_run < triggered_date:
                     next_run = triggered_date
+                alarm['events'] = esm.list_alarm_events(alarm['ID'])
```

Comment:
@yaakovi 
Is it possible that the query for the alarm will return nothing?
If so, will it return just an empty dict without error?

Reply:
no alarms = empty list
error = raising an exception